src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -532,7 +549,7 @@ public function resolve($file)
     protected function resolvePath(string $path)
     {
         $class = $this->getMigrationClass($this->getMigrationName($path));
-
+//        dd($class, $this->getMigrationName($path));
```

Comment:
Debug artifact should be cleaned up and removed here, not just commented out :smile:

Reply:
lolz, on it.